layouts/index.html

```diff
@@ -3,22 +3,48 @@
     {{ $style := resources.Get "sass/main.scss" | resources.ToCSS }}
     {{ $cookieStyles := resources.Get "sass/cookie.scss" | resources.ToCSS }}
     <title>{{ print .Site.Title }}</title>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
     <link rel="stylesheet" href="{{ $style.RelPermalink }}" />
     <link rel="stylesheet" href="{{ $cookieStyles.RelPermalink }}">
     <link rel="icon" href="/images/logo.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
-  <body>
+  <body>    
     <div id="osuButtonContainer">
       <div id="buttonWrapper">
-        <osuButton @osubtnclicked="showOptions" @osubtnclear="clearOptions" v-bind:src="src"></osuButton>
-        <div id="allOptions">
-          <osuOption ref="optionsMenu" v-for="option in options" v-bind:filename="option.filename" v-bind:url="option.url">
-          </osuOption>
+        <div id="logoContainer">
+            <img id="logo" src="/images/logo.png" draggable="false"></img>
+        </div>
+        <div id="optionsContainer">
+          <div class="option">
+            <a class="optionImage" href="/about">
+              <img class="optionImage" src="images/menu-button-1.png"/>
+            </a>
+          </div>
+          <div class="option">
+            <a class="optionImage" href="/members">
+              <img class="optionImage" src="images/menu-button-2.png"/>
+            </a>
+          </div>
+          <div class="option">
+            <a class="optionImage" href="/events">
+              <img class="optionImage" src="images/menu-button-3.png"/>
+            </a>
+          </div>
+          <div class="option">
+            <a class="optionImage" href="https://discord.gg/mfWqAFg">
```

Comment:
I think this `href` can be changed to use `{{ $.Site.Params.urls.discord }}` since we have a site parameter for that now.